common/src/main/java/com/google/udmi/util/SheetsAppender.java

```diff
@@ -0,0 +1,36 @@
+package com.google.udmi.util;
+
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.AppenderBase;
+import ch.qos.logback.core.encoder.Encoder;
+
+public class SheetsAppender extends AppenderBase<ILoggingEvent> {
+
+  private static SheetsOutputStream sheetsOutputStream;
+  private Encoder<ILoggingEvent> encoder;
+
+  @Override
+  protected void append(ILoggingEvent eventObject) {
```

Comment:
Where is this used? I'm trying to evaluate the wisdom of using the `ch.qos.` package... and I can't quite tell how it's being used!